### PR TITLE
My adventure

### PR DESCRIPTION
**Pull request by Ankit-Dhyani**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2552)

### CODE REVIEW

File: hindi/bollywood.md [hunk 1]
```diff
@@ -16,4 +16,6 @@ Amithab, [go here](actor3/amithab.md)
 दीपिका पादुकोन  के लिए
 [ यहाँ  ](./actor5/deepika.md)
 
-Art of Living [Know More](art-of-living/art-of-living.md)
\ No newline at end of file
+Art of Living [Know More](art-of-living/art-of-living.md)
+
+[Sunny Deol](sunny_deol/sunny.md)
```
Comment (casey-collab) [inline hindi/bollywood.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: hindi/Sunny_Deol/sunny.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Sunny Deol is eldest son of [Dharmendra](https://en.wikipedia.org/wiki/Dharmendra).
```
Comment (casey-collab) [inline hindi/Sunny_Deol/sunny.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Ankit-Dhyani** [comment]
Changes made. Please review this agin, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :sunglasses:

**Pull request merged**